guillotina/tests/test_attachment.py

```diff
@@ -27,6 +29,33 @@
     ),
 ]
 
+@pytest.mark.parametrize("manager_type", _pytest_params)
+async def test_db_file_storage(manager_type, redis_container, container_requester):
+    async with container_requester as requester:
+        response, status = await requester(
+            "POST",
+            "/db/guillotina/",
+            data=json.dumps({"@type": "Item", "@behaviors": [IAttachment.__identifier__], "id": "foobar"}))
+        assert status == 201
+        response, status = await requester(
+            "PATCH",
+            "/db/guillotina/foobar/@upload/file",
+            data=b"X" * 1024 * 1024 * 10,
+            headers={"x-upload-size": str(1024 * 1024 * 10)},
+        )
+        assert status == 200
+        request = utils.get_mocked_request(db=requester.db)
+        root = await utils.get_root(request)
+        container = await root.async_get("guillotina")
+        obj = await container.async_get("foobar")
+        behavior = IAttachment(obj)
+        await behavior.load()
+        file_manager = FileManager(obj, request, behavior.file)
```

Comment:
Right here, `behavior.file` is wrong.

It should be `IAttachment['file'].bind(behavior)`

Reply:
Yeap, you are right! thanks for pointing that out. I am closing this